src/pages/workspaces/workspace/WorkspaceMenu.js

```diff
@@ -47,7 +47,14 @@ const WorkspaceMenu = ({
 };
 
 const DynamicWorkspaceMenuContent = ({ namespace, name, onClone, onShare, onDelete, onLock, onLeave }) => {
-  const { workspace } = useWorkspaceDetails({ namespace, name }, ['accessLevel', 'canShare', 'workspace.cloudPlatform', 'workspace.isLocked']);
+  const { workspace } = useWorkspaceDetails({ namespace, name }, [
+    'accessLevel',
+    'policies',
+    'canShare',
+    'workspace.cloudPlatform',
```

Comment:
I think that cloudPlatform can be dropped (we used to disable some menu items for Azure workspaces).